### PR TITLE
[TASK] Update hints of Extension Builder

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See TYPO3-Documentation/T3DocTeam#174 for details.

### CODE REVIEW

File: Documentation/Settings.cfg [hunk 1]
```diff
@@ -1,41 +1,16 @@
 # coding: utf-8
 
-# #####
```
Comment (linawolf) [inline Documentation/Settings.cfg:3]: Changes like these are out of scope of the PR

File: Documentation/ApiOverview/Configuration/Index.rst [hunk 2]
```diff
@@ -35,25 +35,13 @@ you will be directed to another manual or to a different section in this manual.
    ConfigurationSyntax
    ConfigurationMethods
    ../TypoScriptSyntax/Index
-   ../Yaml/Index
```
Comment (linawolf) [inline Documentation/ApiOverview/Configuration/Index.rst:38]: Changes like these are out of scope

**Review by linawolf — CHANGES_REQUESTED** [review]

File: Documentation/Settings.cfg (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Settings.cfg:3]: A separate commit should be enough, not?

File: Documentation/ApiOverview/Configuration/Index.rst (hunk 2 above)
Comment (alexander-nitsche) [reply Documentation/ApiOverview/Configuration/Index.rst:38]: A separate commit should be enough, not?

**Review by linawolf — APPROVED** [review]
